app/views/appointments/calendar.html.erb

```diff
@@ -3,26 +3,31 @@
     <h1>Agenda</h1>
     <%= link_to "Nueva cita", new_appointment_path, class:"btn btn-outline-secondary btn-sm" %>
     <%= week_calendar(events: @appointments) do |date, appointments| %>
-      <%= date.day %>
-
-      <% appointments.each do |appointment| %>
-        <div class="event">
-          <div class="card-header">
-            <%= link_to appointment.title, appointment_path(appointment) %>
-          </div>
-          <div class="card-body">
-            <p>
-              <%=appointment.description%>
-            </p>
-          </div>
-          <div class="cars-footer">
-            <p class="card-text">
-              <%= appointment.start_time.strftime("%I:%M %p") if appointment.start_time %>
-              <%= appointment.end_time.strftime("- %I:%M %p") if appointment.end_time %>
-            </p>
-          </div>
+      <%= puts date %>
+      <div class="row">
+        <div class="col">
+          <% (8..20).each do |hour| %>
+            <div>
+              <strong><%= "%02d:00" % hour %></strong>
+              <% if appointments.any? { |appointment| appointment.start_time.hour == hour } %>
+              <% appointments.select { |appointment| appointment.start_time.hour == hour }.each do |appointment| %>
+                  <div class="event">
+                    <div class="card-header">
+                      <%= link_to appointment.title, appointment_path(appointment), class: "badge badge-success" %>
+                    </div>
+                  </div>
+                <% end %>
+              <% else %>
+                <div class="event">
+                  <div class="card-header">
+                    <%= link_to "Nueva cita", new_appointment_path(selected_date: "#{date} #{hour}"), class: "btn btn-primary" %>
+                  </div>
+                </div>
+              <% end %>
+            </div>
+          <% end %>
```

Comment:
Are you mixing up PRs?

Reply:
yes

Reply:
It was a mistake but I already reverted the pr changes